### PR TITLE
Documentations & Test Coverage

### PR DESCRIPTION
**Pull request by ykamber99**
### Documentations

1. Chinese version of the Readme
2. Code documentation for royalty split
### Test Coverage
Test for requirement statement and if statement in PaymentModule and RoyaltyModule

### CODE REVIEW

File: contracts/PaymentModule.sol [hunk 1]
```diff
@@ -132,6 +143,18 @@ contract PaymentModule is StorageStructure, Ownable {
         } else return 0;
     }
 
+    function checkRegisterPayment(
```
Comment (Therecanbeonlyone1969) [inline contracts/PaymentModule.sol:146]: @ykamber99 do you want to check if a payment was registered? If yes, I would use a boolean variable as the return value. True if the payment is registered, and false if not.

**Review by Therecanbeonlyone1969 — CHANGES_REQUESTED** [review]
See my comment on the `checkRegisterPayment` function.

File: contracts/PaymentModule.sol (hunk 1 above)
Comment (ykamber99) [reply contracts/PaymentModule.sol:146]: Yes!

Comment (Therecanbeonlyone1969) [reply contracts/PaymentModule.sol:146]: @ykamber99 then make that little code change and i will merge

Comment (Therecanbeonlyone1969) [reply contracts/PaymentModule.sol:146]: and update the test too, please

Comment (ykamber99) [reply contracts/PaymentModule.sol]: I have updated the function and added a related test.

**Review by Therecanbeonlyone1969 — APPROVED** [review]
Lgtm now

**Pull request merged**